javascript/javascript3/week1/giphyApi/giphyApi.js

```diff
@@ -0,0 +1,45 @@
+/* Giphy api */
+
+const gifButton = document.getElementById("button1");
+
+const yourSearh = document.getElementById("gifSearch");
+
+const showGif = document.getElementById("number");
+
+const image = document.getElementById("img");
+
+const searh = () => {
+  const key = "xj7oXM8sQY98cMeuuWPPBI3zecV6wNYo";
+  const inputSearch = yourSearh.value;
+  let howMany = showGif.value;
+
+  fetch(
+    `https://api.giphy.com/v1/gifs/search?api_key=${key}&limit=${howMany}&offset=0&rating=G&lang=en&q=${inputSearch}`
+  )
+    .then(response => response.json())
+    .then(data => {
```

Comment:
please use a callback function! don't put all the functionality of your app inside of the `then()`